source/features/patch-diff-links.tsx

```diff
@@ -29,5 +29,8 @@ void features.add({
 	include: [
 		pageDetect.isCommit
 	],
+	exclude: [
+		() => document.title.startsWith('Commit range not found · Pull Request')
```

Comment:
Can you also send a PR for this in github-url-detection?

As `isPRCommit404`

Reply:
Done: https://github.com/fregante/github-url-detection/pull/12